example/app.config.js

```diff
@@ -33,4 +33,9 @@ export const app_config = {
    * can be rewritten for any request using props numberElementsByPage
    */
   defaultRequestLimit: 10,
+
+  /*
+   * This setting is used to define the default mode of session connection
+   */
```

Comment:
A bit more explanation could be nice 

"This configuration allows you to activate or not the connection sessions. If enabled, the user will be able to save sessions on their device for later use."